src/export/util.ts

```diff
@@ -0,0 +1,41 @@
+// (C) 2019 GoodData Corporation
+import { DashboardExport } from "@gooddata/typings";
+import { BAD_REQUEST_STATUS, SUCCESS_REQUEST_STATUS } from "../constants/errors";
+
+export function isDataExported(responseHeaders: Response): boolean {
+    const taskState = responseHeaders.status;
+    return taskState === SUCCESS_REQUEST_STATUS || taskState >= BAD_REQUEST_STATUS; // OK || ERROR
+}
+
+export function isDateFilter(
+    filter: DashboardExport.FilterContextItem,
+): filter is DashboardExport.IDateFilter {
+    if ((filter as DashboardExport.IDateFilter).dateFilter) {
+        return true;
+    }
+    return false;
+}
+
+export function sanitizeDateFilter(filter: DashboardExport.IDateFilter): DashboardExport.IDateFilter {
+    const { dateFilter } = filter;
+    return {
+        dateFilter: {
+            ...dateFilter,
+            from: String(dateFilter.from),
```

Comment:
this does not looks right to me from design point of view... is this really necessary? 

Also as Absolute and Relative date filters are using different properties it should be separated as we already using (AFM, VisObj) into two independent definitions and not like this... is API architect aware of this ? (@lupko)